templates/base.html

```diff
@@ -38,7 +38,8 @@
     <link rel="icon" type="image/png" href="{{ url_for('static', filename='img/favicon/favicon-16x16.png') }}" sizes="16x16">
     <link rel="manifest" href="{{ url_for('static', filename='img/favicon/manifest.json') }}">
     <link rel="mask-icon" href="{{ url_for('static', filename='img/favicon/safari-pinned-tab.svg') }}" color="#5bbad5">
-    <link rel="shortcut icon" href="{{ url_for('static', filename='img/favicon/favicon.ico') }}">
+    <!-- favicon changes depending on server type -->
+    <link rel="shortcut icon" href="{{ url_for('static', filename=config.FAVICON.get(config.SERVER_TYPE))  }}">
```

Comment:
We also should consider that by [default](https://www.washington.edu/accesscomputing/webd2/student/unit4/module3/lesson4.html#:~:text=By%20default%2C%20browsers%20will%20look%20for%20a%20file,file%20in%20the%20%3Chead%3E%20section%20of%20your%20page.), browsers will look for a file titled "favicon.ico" in the root directory of the web site to be used as a favicon.

Should we also keep favicon.ico as a default, just in case, for this purpose?

Reply:
With the recent change there should be a favicon.ico for each platform. Did you want a more general one just in case?

Reply:
Yes, keeping favicon.ico as default.